libvips/conversion/smartcrop.c

```diff
@@ -434,6 +450,21 @@ vips_smartcrop_class_init( VipsSmartcropClass *class )
 		G_STRUCT_OFFSET( VipsSmartcrop, interesting ),
 		VIPS_TYPE_INTERESTING, VIPS_INTERESTING_ATTENTION );
 
+	VIPS_ARG_INT( class, "attention_x", 2, 
+		_( "AttentionX" ), 
+		_( "Horizontal position of attention center" ),
```

Comment:
I know this is weird, but libvips uses UK spelling for historical reasons. Could you swap this for `centre`? Same below.